src/lib.rs

```diff
@@ -1,10 +1,17 @@
 #![no_std]
 
-pub type Color = rgb::RGB8;
+pub use {rgb::RGB16, rgb::RGB8, rgb::RGBA};
```

Comment:
maybe re-export `rgb:RGB` as well to keep both generic `RGB` and `RGBW` accessible in the same way?

Reply:
Good catch